src/components/Profile/ProfileNavigation.tsx

```diff
@@ -20,15 +21,18 @@ export const ProfileNavigation = ({ username }: ProfileNavigationProps) => {
   const { data: userOverview } = trpc.userProfile.overview.useQuery({
     username,
   });
-  const activePath = router.pathname.split('/').pop() || 'profile';
+  const overviewPath = '[username]';
```

Comment:
This can be declared outside the component so it doesn't gets declared everytime the component re-renders